README.rst

```diff
@@ -13,10 +13,10 @@ The project is a split of ``astropy._erfa`` module, developed in the
 context of Astropy_ project, into a standalone package.  It contains
 the ERFA_ C source code as a git submodule.
 
-.. _Python: https://www.python.org/
 .. _ERFA: https://github.com/liberfa/erfa
 .. _Numpy: https://numpy.org/
 .. _Astropy: https://www.astropy.org
+.. _Python: https://www.python.org
```

Comment:
Any particular reason to swap this around?

Reply:
Linkcheck exits with error because in `docs/quicklook.rst` there is a reference to `Python_` but the target is not included in the same file.
I disovered it making local testing.

Reply:
Ah, yes, and since you cut on astropy, now it works. OK!